File: Eulerpl/Euler6.pl

#!/usr/bin/perl
use strict;
use warnings;
use 5.010;

my $sumofsquares = 0;
my $squareofsums = 0;
my $num = 0;
for(1..100){
    $squareofsums += $_;
    $sumofsquares += ($_ * $_);
}

$squareofsums *= $squareofsums;

$num = $squareofsums - $sumofsquares;

say $num;

File: Eulerpy/Euler6.py
#!/usr/bin/python
sumofsquares = 0
squareofsums = 0
for i in xrange(101):
	squareofsums += i
sqaureofsums = squareofsums * squareofsums

for i in xrange(101):
	sumofsquares += (i*i)

num = sqaureofsums - sumofsquares

print num 
